service/src/test/java/bio/terra/landingzone/library/landingzones/definition/factories/CromwellBaseResourcesFactoryTest.java

```diff
@@ -40,24 +39,26 @@ void setUp() {
   @Test
   void deploysLandingZoneV1_resourcesAreCreated() throws InterruptedException {
     String landingZoneId = UUID.randomUUID().toString();
-    var resources =
-        landingZoneManager
-            .deployLandingZoneAsync(
-                landingZoneId,
-                CromwellBaseResourcesFactory.class.getSimpleName(),
-                DefinitionVersion.V1,
-                null)
-            .collectList()
-            .block();
-
-    // Note that this resource list does not include pre-requisite resources
-    assertThat(resources, hasSize(6));
```

Comment:
I removed the assert on whatever `deployLandingZoneAsync` returns because it is kind of irrelevant given it is only the resources that happen to be created in the last step.